Correction to schema validation for ADM0

dgreenslade opened the pull request:
The validation schema for AMD0 layer was causing the transform dags to fail.

dgreenslade committed:
Issue with schema validation for ADM9

geojayuu commented:
Commit pulled and tested on local AirFlow. Data uploaded to GoCD for checking through MapChef.

geojayuu reviewed (commented):
Seems fine. Commit pulled and tested on local AirFlow. Data uploaded to G:Drive for GoCD MapChef tests.

jwharfe committed:
Ammended ADM1 schema as per the schema in the CMF:
GIS/5_Data_schema/admin1_py.yml
Ammended Yemen schema to account for column names in source.

geojayuu commented:
@dgreenslade @egozverev As discussed changes to comply with the CMF schema (ie. https://github.com/mapaction/default-crash-move-folder/blob/9774039fdc15855b64248467e22a0480af35bd9d/20YYiso3nn/GIS/5_Data_schemas/admin1_py.yml) have been added for a default schema mapping (AMD1)
Yemen, which uses an override has also been amended.
Similar changes will be required depending on the level of Admin that we wish to go to but up to now only ADM0/1 are included in the AirFlow dags.

geojayuu committed:
Merge branch 'master' into fix-schema-validation

geojayuu committed:
Update config_parser.py

geojayuu committed:
Update config_parser.py
